code/datums/browser.dm

```diff
@@ -390,7 +390,7 @@
 			if ("color")
 				settings["mainsettings"][setting]["value"] = input(user, "Enter new value for [settings["mainsettings"][setting]["desc"]]", "Enter new value for [settings["mainsettings"][setting]["desc"]]", settings["mainsettings"][setting]["value"]) as color
 			if ("boolean")
-				settings["mainsettings"][setting]["value"] = input(user, "[settings["mainsettings"][setting]["desc"]]?") in list("Yes","No")
+				settings["mainsettings"][setting]["value"] = (settings["mainsettings"][setting]["value"] == "Yes") ? "No" : "Yes"
```

Comment:
This doesn't look right. Why is `== "Yes"` evaluating to No?

Reply:
Because that's the toggle action, if it's yes, it becomes no, if it's no, it becomes yes. I can't do it with a negation operator because they're the actual string words